jekyll_pages/_config.yml

```
@@ -0,0 +1,53 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: SensePitch Edge
+email: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  reverse proxy designed to enhance the availability and performance of web applications. using Java, Netty, and BoringSSL. ingress controller to protect against malicious traffic, DoS attacks, improve site delivery performance
+baseurl: "/sensepitch-edge" # the subpath of your site, e.g. /blog
+url: "https://globalworming.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

```suggestion
baseurl: "/" # the subpath of your site, e.g. /blog
url: "https://sensepitch.io" # the base hostname & protocol for your site, e.g. http://example.com
```

adjust the config as needed